components/match_ticker/commons/match_ticker_custom.lua

```diff
@@ -45,6 +45,15 @@ function CustomMatchTicker.mainPage(frame)
 	return MatchTicker(args):query():create()
 end
 
+---Entry point for display on the main page with the new style
+---@param frame Frame?
+---@return Html
+function CustomMatchTicker.newMainPage(frame)
```

Comment:
is the new version only intended for main page or will it be used on tournament pages etc pp too eventually?

Reply:
For now it will only be used for main page but it will most likely be expanded to other pages in the future